src/main/kotlin/com/atlassian/performance/tools/jiraactions/page/IssueCreateDialog.kt

```diff
@@ -49,12 +54,32 @@ internal class IssueCreateDialog(
     }
 
     fun fillRequiredFields(): IssueCreateDialog {
+
+        driver.wait(elementToBeClickable(configColumnField)).click()
+        val locator = By.xpath("//div[@class='qf-picker-header']/dl/dd/a[text()='All']")
+        if (driver.isElementPresent(locator)) {
+            driver.wait(elementToBeClickable(locator)).click()
+        } else {
+            driver.wait(elementToBeClickable(configColumnField)).click()
+        }
         form.fillRequiredFields()
+        selectResolution()
```

Comment:
Why doesn't the `form.fillRequiredFields()` cover that already? Is the resolution required? If so, why didn't we detect it? It'd be much easier to reason about if we had a red test that shows the need for this change.